test/onnx/test_onnx_export.py

```diff
@@ -19,27 +20,73 @@
 sys.path.append(pytorch_test_dir)
 
 
+def export_to_onnx(
+    model: Union[torch.nn.Module, torch.jit.ScriptFunction],
+    input: Tuple[torch.Tensor],
+    custom_ops: Optional[
+        Iterable[
+            Tuple[
+                str,
+                Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
+                int,
+            ]
+        ]
+    ] = None,
+    mocks: Optional[Iterable[unittest.mock.patch]] = None,
+    operator_export_type: OperatorExportTypes = OperatorExportTypes.ONNX,
+    opset_version=torch.onnx.symbolic_helper._export_onnx_opset_version,
+) -> onnx.ModelProto:
+    """Exports `model(input)` to ONNX and returns it.
+
+    Custom operators and/or unittest patches can be used help reproducing specific behaviors.
+
+    Args:
+        model: model to export
+        input: model input with same format as `torch.onnx.export(..,args,...)`
+        custom_ops: list of custom operators to use during export
+        mocks: list of mocks to use during export
+        operator_export_type: export type as described by `torch.onnx.export(...operator_export_type,...)`
+        opset_version: ONNX opset version as described by `torch.onnx.export(...opset_version,...)`
+    Returns:
+        A valid ONNX model (`onnx.ModelProto`)
+    """
+    custom_ops = custom_ops or []
```

Comment:
I am still not sure about the type for custom_ops. If the type were

```
Iterable[
            Tuple[
                str,
                Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
                int,
            ]
        ]
```

unpacking yields

```
Tuple[
    str,
    Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
    int,
]
```

I don't think tuple implements `__enter__` and `__exit__`? What is the return type of the `custom_op` function?